src/main/java/uk/sky/cqlmigrate/CqlMigratorConfig.java

```diff
@@ -0,0 +1,64 @@
+package uk.sky.cqlmigrate;
+
+import com.datastax.driver.core.ConsistencyLevel;
+import com.google.common.base.Preconditions;
+
+class CqlMigratorConfig {
+    private final CassandraLockConfig cassandraLockConfig;
+    private final ConsistencyLevel readConsistencyLevel;
+    private final ConsistencyLevel writeConsistencyLevel;
+
+    private CqlMigratorConfig(CassandraLockConfig cassandraLockConfig, ConsistencyLevel readConsistencyLevel, ConsistencyLevel writeConsistencyLevel) {
+        Preconditions.checkNotNull(cassandraLockConfig);
+        Preconditions.checkNotNull(readConsistencyLevel);
+        Preconditions.checkNotNull(writeConsistencyLevel);
+
+        this.cassandraLockConfig = cassandraLockConfig;
+        this.readConsistencyLevel = readConsistencyLevel;
+        this.writeConsistencyLevel = writeConsistencyLevel;
+    }
```

Comment:
`checkNotNull` can be inlined, e.g. `this.cassandraLockConfig = checkNotNull(cassandraLockConfig)`